pkgs/tools/misc/rpm-ostree/default.nix

```diff
@@ -60,7 +114,7 @@ stdenv.mkDerivation rec {
 
   meta = with stdenv.lib; {
     description = "A hybrid image/package system. It uses OSTree as an image format, and uses RPM as a component model";
-    homepage = https://rpm-ostree.readthedocs.io/en/latest/;
+    homepage = "https://rpm-ostree.readthedocs.io/en/latest/";
```

Comment:
I just want to note this package is nowhere in the dependency tree, and @copumpkin hasn't contributed updates to it for a long while https://github.com/NixOS/nixpkgs/commit/f35ab9c2f814e88986ce8ed2b21728d96e71b784#diff-201e050d13988f27cdd42d11e1d9979c. Do you have any use for it? I certainly don't and don't really care to continue maintaining it.

Reply:
Last time I talked to them, they told me they no longer use it. But it still felt like an useful thing to keep, if only to have more coverage to see if ostree broke.